### ISSUE TITLE
Remove and uninstall commands completely freezing my computer

### ISSUE THREAD

**Issue by shivendra90** [conda/conda#10510]
## Current Behavior

By default conda has installed a very old version of Node (6.13.1) and I thought to clean reinstall it either through the conda-forge channel or from the snap store since I'm using Linux. However, simply running the command `conda uninstall nodejs` takes forever to solve the environment and eventually freezes my system in the process. 

Not just with node but removing any library/app completely freezes the system.

### Steps to Reproduce

Simply run:

```
conda uninstall nodejs
conda remove jupyterlab-plotly-extension
```

This is only with `uninstall` and `remove` commands.

## Expected Behavior

Node is not that of a big deal and the process should continue and the environment be solved.

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
active environment : base
    active env location : /home/shivendra/anaconda3
            shell level : 1
       user config file : /home/shivendra/.condarc
 populated config files : /home/shivendra/.condarc
          conda version : 4.9.2
    conda-build version : 3.20.3
         python version : 3.7.9.final.0
       virtual packages : __glibc=2.31=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/shivendra/anaconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/shivendra/anaconda3/pkgs
                          /home/shivendra/.conda/pkgs
       envs directories : /home/shivendra/anaconda3/envs
                          /home/shivendra/.conda/envs
               platform : linux-64
             user-agent : conda/4.9.2 requests/2.25.1 CPython/3.7.9 Linux/5.4.0-65-generic ubuntu/20.04.2 glibc/2.31
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False

```

</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
==> /home/shivendra/.condarc <==
ssl_verify: True
channels:
  - conda-forge
  - defaults


```

</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
# packages in environment at /home/shivendra/anaconda3:
#
# Name                    Version                   Build  Channel
_ipyw_jlab_nb_ext_conf    0.1.0                    py37_0    defaults
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                      1_llvm    conda-forge
_pytorch_select           0.2                       gpu_0    defaults
_tflow_select             2.3.0                       mkl    defaults
absl-py                   0.11.0             pyhd3eb1b0_1    defaults
alabaster                 0.7.12                   py37_0    defaults
anaconda-client           1.7.2                    py37_0    defaults
anaconda-navigator        1.10.0                   py37_0    defaults
anyio                     2.0.2                    pypi_0    pypi
anyqt                     0.0.10                   py37_0    defaults
argh                      0.26.2                   py37_0    defaults
argon2-cffi               20.1.0           py37h7b6447c_1    defaults
astor                     0.8.1            py37h06a4308_0    defaults
astroid                   2.4.2                    py37_0    defaults
async_generator           1.10             py37h28b3542_0    defaults
atomicwrites              1.4.0                      py_0    defaults
attrs                     20.3.0             pyhd3eb1b0_0    defaults
autopep8                  1.5.5              pyh44b312d_0    conda-forge
babel                     2.9.0              pyhd3eb1b0_0    defaults
backcall                  0.2.0              pyhd3eb1b0_0    defaults
backports                 1.0                pyhd3eb1b0_2    defaults
backports.functools_lru_cache 1.6.1              pyhd3eb1b0_0    defaults
backports.tempfile        1.0                pyhd3eb1b0_1    defaults
backports.weakref         1.0.post1                  py_1    defaults
baycomp                   1.0.2                      py_1    defaults
beakerx                   1.4.1            py37hc8dfbb8_3    conda-forge
beakerx_kernel_autotranslation 2.0.0                    py37_1    beakerx
beautifulsoup4            4.9.3              pyha847dfd_0    defaults
blas                      1.0                         mkl    defaults
bleach                    3.3.0              pyhd3eb1b0_0    defaults
bokeh                     2.2.3                    py37_0    defaults
bottle                    0.12.18                    py_0    conda-forge
bottleneck                1.3.2            py37heb32a55_1    defaults
brotlipy                  0.7.0           py37h27cfd23_1003    defaults
bzip2                     1.0.8                h7b6447c_0    defaults
c-ares                    1.17.1               h27cfd23_0    defaults
ca-certificates           2020.12.5            ha878542_0    conda-forge
cachecontrol              0.12.6                     py_0    defaults
cairo                     1.16.0            h18b612c_1001    conda-forge
cairocffi                 1.1.0                    pypi_0    pypi
cairosvg                  2.4.2                    pypi_0    pypi
catalogue                 1.0.0                    py37_1    defaults
certifi                   2020.12.5        py37h89c1867_1    conda-forge
cffi                      1.14.4           py37h261ae71_0    defaults
chardet                   4.0.0           py37h06a4308_1003    defaults
click                     7.1.2              pyhd3eb1b0_0    defaults
cloudpickle               1.6.0                      py_0    defaults
clyent                    1.2.2                    py37_1    defaults
cmocean                   2.0                      pypi_0    pypi
commonmark                0.9.1                      py_0    defaults
conda                     4.9.2            py37h89c1867_0    conda-forge
conda-build               3.20.3                   py37_0    defaults
conda-env                 2.6.0                         1    defaults
conda-package-handling    1.7.2            py37h03888b9_0    defaults
conda-verify              3.4.2                      py_1    defaults
convertdate               2.2.0                    pypi_0    pypi
cryptography              3.3.1            py37h3c74f83_0    defaults
cssselect                 1.1.0                    pypi_0    pypi
cssselect2                0.2.2                    pypi_0    pypi
cudatoolkit               10.0.130                      0    defaults
cudnn                     7.6.5                cuda10.0_0    defaults
cycler                    0.10.0                   py37_0    defaults
cymem                     2.0.4            py37h2531618_0    defaults
cython-blis               0.4.1            py37h7b6447c_1    defaults
cytoolz                   0.11.0           py37h7b6447c_0    defaults
dask                      2021.1.1           pyhd3eb1b0_0    defaults
dask-core                 2021.1.1           pyhd3eb1b0_0    defaults
dask-glm                  0.2.0                    py37_0    defaults
dask-ml                   1.8.0              pyhd3eb1b0_0    defaults
dataclasses               0.7                      py37_0    defaults
dataframe-image           0.1.1                    pypi_0    pypi
dbus                      1.13.18              hb2f20db_0    defaults
decorator                 4.4.2              pyhd3eb1b0_0    defaults
defusedxml                0.6.0              pyhd3eb1b0_0    defaults
dictdiffer                0.8.0                      py_0    defaults
diff-match-patch          20200713                   py_0    defaults
distributed               2021.1.1         py37h06a4308_1    defaults
docutils                  0.16                     py37_1    defaults
eli5                      0.10.1                   py37_1    conda-forge
entrypoints               0.3                      py37_0    defaults
et_xmlfile                1.0.1                   py_1001    defaults
expat                     2.2.10               he6710b0_2    defaults
factor-analyzer           0.3.2                    pypi_0    pypi
fastai                    1.0.60                        1    fastai
fastprogress              1.0.0                      py_0    conda-forge
filelock                  3.0.12             pyhd3eb1b0_1    defaults
flake8                    3.8.4                      py_0    defaults
fontconfig                2.13.1            hba837de_1004    conda-forge
freetype                  2.10.4               h5ab3b9f_0    defaults
fribidi                   1.0.10               h7b6447c_0    defaults
fsspec                    0.8.3                      py_0    defaults
future                    0.18.2                   py37_1    defaults
gast                      0.2.2                    py37_0    defaults
gettext                   0.19.8.1             h9b4dc7a_1    defaults
glib                      2.66.1               h92f7085_0    defaults
glob2                     0.7                pyhd3eb1b0_0    defaults
google-pasta              0.2.0                      py_0    defaults
graphite2                 1.3.14               h23475e2_0    defaults
graphviz                  2.42.3               h0511662_0    conda-forge
grpcio                    1.31.0           py37hf8bcb03_0    defaults
gst-plugins-base          1.14.0               h8213a91_2    defaults
gstreamer                 1.14.0               h28cd5cc_2    defaults
h11                       0.9.0                      py_0    defaults
h2                        4.0.0                    py37_0    defaults
h5py                      2.10.0           py37h7918eee_0    defaults
harfbuzz                  2.4.0                hca77d97_1    defaults
hdf5                      1.10.4               hb1b8bf9_0    defaults
heapdict                  1.0.1                      py_0    defaults
hide-code                 0.6.0                    pypi_0    pypi
holidays                  0.10.1                   pypi_0    pypi
hpack                     3.0.0            py37h28b3542_0    defaults
httpcore                  0.12.3             pyhd3eb1b0_0    defaults
httpx                     0.16.1                     py_0    defaults
hyperframe                5.2.0                      py_0    defaults
icu                       58.2                 he6710b0_3    defaults
idna                      2.10               pyhd3eb1b0_0    defaults
imagesize                 1.2.0              pyhd3eb1b0_0    defaults
importlib-metadata        2.0.0                      py_1    defaults
importlib_metadata        2.0.0                         1    defaults
iniconfig                 1.1.1              pyhd3eb1b0_0    defaults
intel-openmp              2020.2                      254    defaults
intervaltree              3.1.0                      py_0    defaults
ipykernel                 5.4.3            py37h888b3d9_0    conda-forge
ipython                   7.20.0           py37hb070fc8_1    defaults
ipython_genutils          0.2.0              pyhd3eb1b0_1    defaults
ipywidgets                7.6.3              pyhd3eb1b0_1    defaults
isort                     5.7.0              pyhd3eb1b0_0    defaults
jdcal                     1.4.1                      py_0    defaults
jedi                      0.17.1                   py37_0    defaults
jeepney                   0.6.0              pyhd3eb1b0_0    defaults
jinja2                    2.11.3             pyhd3eb1b0_0    defaults
joblib                    1.0.0              pyhd3eb1b0_0    defaults
joypy                     0.2.2                      py_1    conda-forge
jpeg                      9d                   h36c2ea0_0    conda-forge
json5                     0.9.5                      py_0    defaults
jsonschema                3.0.2                    py37_0    defaults
jupyter                   1.0.0                    pypi_0    pypi
jupyter-console           6.2.0                    pypi_0    pypi
jupyter-kite              2.0.2                    pypi_0    pypi
jupyter-server            1.2.2                    pypi_0    pypi
jupyter_client            6.1.7                      py_0    defaults
jupyter_contrib_core      0.3.3                      py_2    conda-forge
jupyter_contrib_nbextensions 0.5.1                    py37_0    conda-forge
jupyter_core              4.7.1            py37h06a4308_0    defaults
jupyter_highlight_selected_word 0.2.0           py37hc8dfbb8_1002    conda-forge
jupyter_latex_envs        1.4.6           py37hc8dfbb8_1001    conda-forge
jupyter_nbextensions_configurator 0.4.1            py37hc8dfbb8_2    conda-forge
jupyter_server            1.3.0            py37h89c1867_0    conda-forge
jupyterlab                3.0.6                    pypi_0    pypi
jupyterlab-kite           2.0.2                    pypi_0    pypi
jupyterlab-plotly-extension 1.0.0                      py_0    conda-forge
jupyterlab-server         2.1.3                    pypi_0    pypi
jupyterlab_pygments       0.1.2                      py_0    defaults
jupyterlab_server         2.2.0              pyhd8ed1ab_0    conda-forge
jupyterlab_widgets        1.0.0              pyhd3eb1b0_1    defaults
keras                     2.3.1                         0    defaults
keras-applications        1.0.8                      py_1    defaults
keras-base                2.3.1                    py37_0    defaults
keras-preprocessing       1.1.2              pyhd3eb1b0_0    defaults
keyring                   22.0.1           py37h06a4308_0    defaults
keyrings.alt              4.0.2            py37h06a4308_0    defaults
kiwisolver                1.3.1            py37h2531618_0    defaults
lazy-object-proxy         1.4.3            py37h27cfd23_2    defaults
lcms2                     2.11                 h396b838_0    defaults
ld_impl_linux-64          2.35.1               hea4e1c9_1    conda-forge
lesscpy                   0.14.0                   pypi_0    pypi
libarchive                3.4.2                h62408e4_0    defaults
libblas                   3.9.0                     8_mkl    conda-forge
libcblas                  3.9.0                     8_mkl    conda-forge
libffi                    3.3                  he6710b0_2    defaults
libgcc                    7.2.0                h69d50b8_2    conda-forge
libgcc-ng                 9.3.0               h5dbcf3e_17    conda-forge
libgfortran-ng            7.3.0                hdf63c60_0    defaults
liblapack                 3.9.0                     8_mkl    conda-forge
liblief                   0.10.1               he6710b0_0    defaults
libllvm10                 10.0.1               hbcb73fb_5    defaults
libpng                    1.6.37               hbc83047_0    defaults
libprotobuf               3.14.0               h8c45485_0    defaults
libsodium                 1.0.18               h7b6447c_0    defaults
libspatialindex           1.9.3                he6710b0_0    defaults
libstdcxx-ng              9.3.0               h2ae2ef3_17    conda-forge
libtiff                   4.1.0                h2733197_1    defaults
libtool                   2.4.6             h7b6447c_1005    defaults
libuuid                   2.32.1            h7f98852_1000    conda-forge
libxcb                    1.14                 h7b6447c_0    defaults
libxml2                   2.9.10               hb55368b_3    defaults
libxslt                   1.1.34               hc22bd24_0    defaults
llvm-openmp               11.0.0               hfc4b9b4_1    conda-forge
llvmlite                  0.34.0           py37h269e1b5_4    defaults
locket                    0.2.1            py37h06a4308_1    defaults
lockfile                  0.12.2           py37h06a4308_0    defaults
lxml                      4.6.2            py37h9120a33_0    defaults
lz4-c                     1.9.3                h2531618_0    defaults
markdown                  3.3.3            py37h06a4308_0    defaults
markupsafe                1.1.1            py37h14c3975_1    defaults
matplotlib                3.3.2                h06a4308_0    defaults
matplotlib-base           3.3.2            py37h817c723_0    defaults
maven                     3.6.0                         0    defaults
mccabe                    0.6.1                    py37_1    defaults
mistune                   0.8.4           py37h14c3975_1001    defaults
mkl                       2020.4             h726a3e6_304    conda-forge
mkl-service               2.3.0            py37he8ac12f_0    defaults
mkl_fft                   1.2.0            py37h23d657b_0    defaults
mkl_random                1.2.0            py37h9fdb41a_1    conda-forge
more-itertools            8.6.0              pyhd3eb1b0_0    defaults
msgpack-python            1.0.2            py37hff7bd54_1    defaults
multipledispatch          0.6.0                    py37_0    defaults
murmurhash                1.0.5            py37h2531618_0    defaults
navigator-updater         0.2.1                    py37_0    defaults
nb-pdf-template           4.0.0                    pypi_0    pypi
nbclassic                 0.2.6              pyhd8ed1ab_0    conda-forge
nbclient                  0.5.2              pyhd8ed1ab_0    conda-forge
nbconvert                 5.6.1                    pypi_0    pypi
nbformat                  5.1.2              pyhd3eb1b0_1    defaults
ncurses                   6.2                  he6710b0_1    defaults
nest-asyncio              1.4.3              pyhd3eb1b0_0    defaults
networkx                  2.5                        py_0    defaults
ninja                     1.10.2           py37hff7bd54_0    defaults
nltk                      3.5                        py_0    defaults
nodejs                    6.13.1                        0    conda-forge
notebook                  6.2.0            py37h06a4308_0    defaults
numba                     0.51.2           py37h04863e7_1    defaults
numexpr                   2.7.2            py37hb2eb853_0    defaults
numpy                     1.20.1           py37haa41c4c_0    conda-forge
numpydoc                  1.1.0              pyhd3eb1b0_1    defaults
nvidia-ml-py3             7.352.0                    py_0    fastai
olefile                   0.46                     py37_0    defaults
openjdk                   8.0.121          zulu8.20.0.5_0    conda-forge
openpyxl                  3.0.6              pyhd3eb1b0_0    defaults
openssl                   1.1.1i               h7f98852_0    conda-forge
opentsne                  0.4.3            py37hfd86e86_1    defaults
opt_einsum                3.1.0                      py_0    defaults
orange-canvas-core        0.1.15                     py_0    defaults
orange-widget-base        4.7.0                      py_1    defaults
orange3                   3.26.0           py37he6710b0_0    defaults
packaging                 20.9               pyhd3eb1b0_0    defaults
pandas                    1.2.1            py37ha9443f7_0    defaults
pandas-profiling          1.4.1                    py37_0    defaults
pandoc                    2.11.0.4             hd18ef5c_0    conda-forge
pandocfilters             1.4.3            py37h06a4308_1    defaults
pango                     1.42.4               h7062337_4    conda-forge
parso                     0.7.0                      py_0    defaults
partd                     1.1.0                      py_0    defaults
patchelf                  0.12                 h2531618_1    defaults
pathlib2                  2.3.5                    pypi_0    pypi
patsy                     0.5.1                    py37_0    defaults
pcre                      8.44                 he6710b0_0    defaults
pdfkit                    0.6.1                    pypi_0    pypi
pexpect                   4.8.0              pyhd3eb1b0_3    defaults
pickleshare               0.7.5           pyhd3eb1b0_1003    defaults
pillow                    8.1.0            py37he98fc37_0    defaults
pip                       20.3.3           py37h06a4308_0    defaults
pixman                    0.38.0               h7b6447c_0    defaults
pkginfo                   1.7.0            py37h06a4308_0    defaults
plac                      1.1.0                    py37_1    defaults
plotly                    4.14.3             pyh44b312d_0    conda-forge
plotly-orca               3.4.2                         0    conda-forge
plotly_express            0.4.1                      py_0    conda-forge
pluggy                    0.13.1                   py37_0    defaults
pmdarima                  1.3.0                    pypi_0    pypi
preshed                   3.0.2            py37he6710b0_1    defaults
prometheus_client         0.9.0              pyhd3eb1b0_0    defaults
prompt-toolkit            3.0.8                      py_0    defaults
protobuf                  3.14.0           py37h2531618_1    defaults
psutil                    5.8.0            py37h27cfd23_1    defaults
ptyprocess                0.7.0              pyhd3eb1b0_2    defaults
py                        1.10.0             pyhd3eb1b0_0    defaults
py-lief                   0.10.1           py37h403a769_0    defaults
py4j                      0.10.9.1                   py_0    defaults
pycodestyle               2.6.0              pyhd3eb1b0_0    defaults
pycosat                   0.6.3            py37h27cfd23_0    defaults
pycparser                 2.20                       py_2    defaults
pydocstyle                5.1.1                      py_0    defaults
pyflakes                  2.2.0              pyhd3eb1b0_0    defaults
pygal                     2.4.0                    pypi_0    pypi
pygments                  2.7.4              pyhd3eb1b0_0    defaults
pylint                    2.6.0                    py37_0    defaults
pymeeus                   0.3.6                    pypi_0    pypi
pynndescent               0.4.8                      py_1    defaults
pyopenssl                 20.0.1             pyhd3eb1b0_1    defaults
pyparsing                 2.4.7              pyhd3eb1b0_0    defaults
pyqt                      5.9.2            py37hcca6a23_4    conda-forge
pyqt5                     5.15.2                   pypi_0    pypi
pyqt5-sip                 12.8.1                   pypi_0    pypi
pyqtgraph                 0.11.0                     py_0    defaults
pyqtwebengine             5.12                     pypi_0    pypi
pyrsistent                0.17.3           py37h7b6447c_0    defaults
pysocks                   1.7.1                    py37_1    defaults
pytest                    6.2.2            py37h06a4308_2    defaults
pytest-runner             5.2                pyhd3eb1b0_0    defaults
python                    3.7.9           hffdb5ce_0_cpython    conda-forge
python-dateutil           2.8.1              pyhd3eb1b0_0    defaults
python-graphviz           0.16               pyhd3eb1b0_1    defaults
python-jsonrpc-server     0.4.0                      py_0    defaults
python-libarchive-c       2.9                pyhd3eb1b0_0    defaults
python-louvain            0.15               pyhd3eb1b0_0    defaults
python_abi                3.7                     1_cp37m    conda-forge
pytorch                   1.3.1           cuda100py37h53c1284_0    defaults
pytz                      2021.1             pyhd3eb1b0_0    defaults
pyxdg                     0.27               pyhd3eb1b0_0    defaults
pyyaml                    5.4.1            py37h27cfd23_1    defaults
pyzmq                     20.0.0           py37h2531618_1    defaults
qdarkstyle                2.8.1                      py_0    defaults
qt                        5.9.7                h5867ecd_1    defaults
qtawesome                 1.0.1                      py_0    defaults
qtconsole                 4.7.7                      py_0    defaults
qtpy                      1.9.0                      py_0    defaults
readline                  8.1                  h27cfd23_0    defaults
regex                     2020.11.13       py37h27cfd23_0    defaults
requests                  2.25.1             pyhd3eb1b0_0    defaults
retrying                  1.3.3                      py_2    conda-forge
rfc3986                   1.4.0                      py_0    defaults
ripgrep                   12.1.1                        0    defaults
rope                      0.18.0                     py_0    defaults
rstcheck                  3.3.1                    pypi_0    pypi
rtree                     0.9.4                    py37_1    defaults
ruamel_yaml               0.15.87          py37h7b6447c_1    defaults
scikit-learn              0.24.1           py37h69acf81_0    conda-forge
scipy                     1.6.0            py37h91f5cce_0    defaults
seaborn                   0.11.0               ha770c72_1    conda-forge
seaborn-base              0.11.0             pyhd8ed1ab_1    conda-forge
secretstorage             3.3.0            py37h06a4308_0    defaults
send2trash                1.5.0              pyhd3eb1b0_1    defaults
serverfiles               0.3.1            py37h06a4308_0    defaults
setuptools                52.0.0           py37h06a4308_0    defaults
singledispatch            3.4.0.3                 py_1001    defaults
sip                       4.19.8           py37hf484d3e_0    defaults
six                       1.15.0           py37h06a4308_0    defaults
sniffio                   1.1.0                    py37_2    defaults
snowballstemmer           2.1.0              pyhd3eb1b0_0    defaults
sortedcontainers          2.3.0              pyhd3eb1b0_0    defaults
soupsieve                 2.1                pyhd3eb1b0_0    defaults
spacy                     2.3.5            py37hff7bd54_0    defaults
sphinx                    3.2.1                      py_0    conda-forge
sphinxcontrib-applehelp   1.0.2              pyhd3eb1b0_0    defaults
sphinxcontrib-devhelp     1.0.2              pyhd3eb1b0_0    defaults
sphinxcontrib-htmlhelp    1.0.3              pyhd3eb1b0_0    defaults
sphinxcontrib-jsmath      1.0.1              pyhd3eb1b0_0    defaults
sphinxcontrib-qthelp      1.0.3              pyhd3eb1b0_0    defaults
sphinxcontrib-serializinghtml 1.1.4              pyhd3eb1b0_0    defaults
spyder                    4.1.5                    py37_0    defaults
spyder-kernels            1.9.4                    py37_0    defaults
sqlite                    3.34.0               h74cdb3f_0    conda-forge
srsly                     1.0.5            py37h2531618_0    defaults
statsmodels               0.12.1           py37h27cfd23_0    defaults
sviewgui                  0.3.5                    pypi_0    pypi
tabulate                  0.8.7                    py37_0    defaults
tbb                       2020.3               hfd86e86_0    defaults
tblib                     1.7.0                      py_0    defaults
tensorboard               2.0.0              pyhb38c66f_1    defaults
tensorflow                2.0.0           mkl_py37h66b46cc_0    anaconda
tensorflow-base           2.0.0           mkl_py37h9204916_0    defaults
tensorflow-estimator      2.0.0              pyh2649769_0    defaults
termcolor                 1.1.0                    py37_1    defaults
terminado                 0.9.2            py37h06a4308_0    defaults
testpath                  0.4.4              pyhd3eb1b0_0    defaults
thinc                     7.4.5            py37h9a67853_0    defaults
threadpoolctl             2.1.0              pyh5ca1d4c_0    defaults
tinycss                   0.4                      pypi_0    pypi
tinycss2                  1.0.2                    pypi_0    pypi
tk                        8.6.10               hbc83047_0    defaults
toml                      0.10.1                     py_0    defaults
toolz                     0.11.1             pyhd3eb1b0_0    defaults
torchvision               0.4.2           cuda100py37hecfc37a_0    defaults
tornado                   6.1              py37h27cfd23_0    defaults
tqdm                      4.56.0             pyhd3eb1b0_0    defaults
traitlets                 4.3.3                    pypi_0    pypi
typed-ast                 1.4.0                    pypi_0    pypi
typing_extensions         3.7.4.3            pyha847dfd_0    defaults
ujson                     4.0.2            py37h2531618_0    defaults
urllib3                   1.26.3             pyhd3eb1b0_0    defaults
wasabi                    0.8.2              pyhd3eb1b0_0    defaults
watchdog                  1.0.2            py37h06a4308_1    defaults
wcwidth                   0.2.5                      py_0    defaults
webencodings              0.5.1                    py37_1    defaults
werkzeug                  0.16.1                     py_0    defaults
wheel                     0.36.2             pyhd3eb1b0_0    defaults
widgetsnbextension        3.5.1                    py37_0    defaults
wrapt                     1.11.2           py37h7b6447c_0    defaults
wurlitzer                 2.0.1                    py37_0    defaults
xgboost                   1.3.3                    pypi_0    pypi
xlrd                      2.0.1              pyhd3eb1b0_0    defaults
xlsxwriter                1.3.7              pyhd3eb1b0_0    defaults
xmltodict                 0.12.0                     py_0    defaults
xorg-kbproto              1.0.7             h7f98852_1002    conda-forge
xorg-libice               1.0.10               h516909a_0    conda-forge
xorg-libsm                1.2.3             h84519dc_1000    conda-forge
xorg-libx11               1.6.12               h516909a_0    conda-forge
xorg-libxext              1.3.4                h516909a_0    conda-forge
xorg-libxpm               3.5.13               h516909a_0    conda-forge
xorg-libxrender           0.9.10            h516909a_1002    conda-forge
xorg-libxt                1.1.5             h516909a_1003    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h7f98852_1002    conda-forge
xorg-xproto               7.0.31            h27cfd23_1007    defaults
xz                        5.2.5                h7b6447c_0    defaults
yaml                      0.2.5                h7b6447c_0    defaults
yapf                      0.30.0                     py_0    defaults
yellowbrick               1.0.1                    py37_0    districtdatalabs
zeromq                    4.3.3                he6710b0_3    defaults
zict                      2.0.0                    pypi_0    pypi
zipp                      3.4.0              pyhd3eb1b0_0    defaults
zlib                      1.2.11               h7b6447c_3    defaults
zstd                      1.4.5                h9ceee32_0    defaults

```

</p></details>

**Comment by grach0v** [comment]
Same problem with `cudatoolkit`

**Comment by criphg** [comment]
Same with bwameth
